output.go

```diff
@@ -3,20 +3,58 @@ package jasper
 import (
 	"io"
 	"io/ioutil"
+	"os"
 
 	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/send"
 	"github.com/pkg/errors"
 )
 
 // OutputOptions provides a common way to define and represent the
 // output behavior of a evergreen/subprocess.Command operation.
 type OutputOptions struct {
-	Output            io.Writer `json:"-"`
-	Error             io.Writer `json:"-"`
-	SuppressOutput    bool      `json:"suppress_output"`
-	SuppressError     bool      `json:"suppress_error"`
-	SendOutputToError bool      `json:"redirect_output_to_error"`
-	SendErrorToOutput bool      `json:"redirect_error_to_output"`
+	Output            io.Writer          `json:"-"`
+	Error             io.Writer          `json:"-"`
+	SuppressOutput    bool               `json:"suppress_output"`
+	SuppressError     bool               `json:"suppress_error"`
+	SendOutputToError bool               `json:"redirect_output_to_error"`
+	SendErrorToOutput bool               `json:"redirect_error_to_output"`
+	Loggers           []Logger           `json:"loggers"`
+	OutputSender      *send.WriterSender `json:"-"`
+	ErrorSender       *send.WriterSender `json:"-"`
```

Comment:
is it possible to make these private? or have them be the send.Sender type and then we do the conversion to writer.Sender internally?